config.yaml.sample

```diff
@@ -210,6 +217,10 @@ toi-prune:
     path: osm
     layer: all
     format: zip
+  # a reduced version of prev s3 entity
```

Comment:
the primary store configuration specifies all entries from `toi-prune:s3` except **layer** and **format**. reduced version is a portion of the store-related configuration that specifies only remaining **layer** and **format**.